services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime, movie_id: int, cinema_hall_id: int):
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: str = None):
+    queryset = MovieSession.objects.all()
```

Comment:
Where are type annotations for returned value?

Reply:
Check lattest commit, please.

Reply:
> Check lattest commit, please.

sry, my bad